contrib/ustreamer/jumpstarter_driver_ustreamer/client.py

```diff
@@ -14,4 +25,64 @@ def state(self):
         return UStreamerState.model_validate(self.call("state"))
 
     def snapshot(self):
+        """
+        Get a snapshot image from the video input
+
+        :return: JPEG image data
+        :rtype: bytes
+        """
         return b64decode(self.call("snapshot"))
+
+    def write_snapshot(self, filename):
```

Comment:
```suggestion
    def write_snapshot(self, filename):
```

I wonder if we should just have a separate module

jumpstarter_imagehash

Providing an adapter to video Inputs:

vhash = ImageHashAdapter(client.video)

vhash.snapshot()
vhash.hash_snapshot()
vhash.assert_snapshot(....)


In this way we'd keep this functionality shared to other potential video drivers, 
and we would keep away imagehash dependency (+scipy, etc..) from ustreamer.

Reply:
Well, this adaptor would require snapshot, we can assume that the video stream is optional :)